src/libsync/connectionvalidator.cpp

```diff
@@ -252,7 +253,18 @@ void ConnectionValidator::slotUserFetched(const QVariantMap &json)
     QString user = json.value("ocs").toMap().value("data").toMap().value("id").toString();
     if (!user.isEmpty()) {
         _account->setDavUser(user);
+
+        AvatarJob *job = new AvatarJob(_account, this);
+        job->setTimeout(20*1000);
```

Comment:
@ogoffart wouldn''t it make more sense to set it in the constructor of AvatarJob anyway?